presentation/src/main/res/layout/activity_main.xml

```diff
@@ -14,5 +16,20 @@
             android:id="@+id/tv_incoming_json"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/incoming_json" />
+            android:text=""
```

Comment:
Not hardcode values, create a empty value in strings.xml

Reply:
done